nextflow.config

```
@@ -1,147 +1,444 @@
-  outdir = './results'
-
-  // Boilerplate options
-  name = false
-  multiqc_config = false
-  email = false
-  email_on_fail = false
-  max_multiqc_email_size = 25.MB
-  plaintext_email = false
-  monochrome_logs = false
-  help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-  tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
+    // Workflow flags
+    outdir = null
+    input = null
+    genome = null
+    fasta = null
 
+    tools = "mobster,viber,pyclone-vi,sparsesignatures,sigprofiler"  // Default Subclonal deconvolution & Signature deconvolution tools
+    publish_dir_mode = "copy"
+    publish_dir_mode_genome_sigprofiler = "move"
+
+    filter = true
+
+    // VEP
+    download_cache_vep = null
+    vep_cache = null
+
+    vep_cache_version = null
+    vep_species = null
+    vep_genome = null
+    vep_custom_args = "--everything --filter_common --per_gene --total_length --offline --format vcf"
+
+    // ANNOTATE_DRIVERS
+    drivers_table = "https://raw.githubusercontent.com/nf-core/test-datasets/refs/heads/tumourevo/data/DRIVER_ANNOTATION/ANNOTATE_DRIVER/Compendium_Cancer_Genes.tsv"
+
+    // vcf2CNAqc
+    vcf_filter_mutations = true
+
+    // CNAqc
+    cnaqc_matching_strategy = "rightmost"
+    cnaqc_karyotypes = 'c("1:0", "1:1", "2:0", "2:1", "2:2")'
+    cnaqc_min_karyotype_size = "0"
+    cnaqc_min_absolute_karyotype_mutations = "100"
+    cnaqc_p_binsize_peaks = "0.005"
+    cnaqc_matching_epsilon = "NULL"
+    cnaqc_purity_error = "0.05"
+    cnaqc_vaf_tolerance = "0.015"
+    cnaqc_n_bootstrap = "1"
+    cnaqc_kernel_adjust = "1"
+    cnaqc_kde = "TRUE"
+    cnaqc_starting_state_subclonal_evolution = "1:1"
+    cnaqc_cluster_subclonal_CCF = "FALSE"
+    cnaqc_muts_per_karyotype = "25"
+    cnaqc_cutoff_QC_PASS = "0.1"
+    cnaqc_method = "ENTROPY"
+    cnaqc_plot_cn = "absolute"
+
+    // join_CNAqc
+    joincnaqc_qc_filter = "FALSE"
+    joincnaqc_keep_original = "TRUE"
+
+    // VAF deconvolution - pyclonevi
+    pyclonevi_n_cluster = 20    // default value for PYCLONEVI
+    pyclonevi_density = 'beta-binomial' // default value for PYCLONEVI
+    pyclonevi_n_grid_point = 100 // default value for PYCLONEVI
+    pyclonevi_n_restarts = 100 // default value for PYCLONEVI
+
+    // VAF deconvolution - mobster
+    mobster_K = "1:5"
+    mobster_tail = "TRUE"
+    mobster_pi_cutoff = "0.02"
+    mobster_n_cutoff = "10"
+    mobster_min_VAF = "0.05"
+
+    // VAF deconvolution - VIBER
+    viber_K = "10"
+    viber_samples = "10"
+    viber_alpha_0 = "1e-06"
+    viber_a_0 = "1"
+    viber_b_0 = "1"
+    viber_maxIter = "5000"
```

Think this one should definitely be changed though.
```suggestion
    viber_max_iter = "5000"
```